### DISCUSSION TITLE
My root domain has expired and it's not available now so I can't renew it. Please tell me how can I access the apps running with this instance.

### DISCUSSION THREAD

**Discussion by amankch** [caprover/caprover#1465] [Q&A]
My root domain has expired and due to some reasons, I wasn't able to renew it. Now I am not able to access any apps running with this instance. Please help me to make it live again.

**Comment by maietta** [comment]
One solution I can think of doesn't involve needing SSH access can be done pretty quickly with minimal effort. It goes as follows:

- You can point your DNS to resolve any domain you want to your server's IP that is running Caprover. For Windows, I use a utility called HostMan, but you can simply edit your hosts file. Once done, all traffic to that domain will hit that IP.
- Then you can go into your Caprover interface and change the domain to a new one.

**Reply by amankch** [reply]
> One solution I can think of doesn't involve needing SSH access can be done pretty quickly with minimal effort. It goes as follows:
> 
> * You can point your DNS to resolve any domain you want to your server's IP that is running Caprover. For Windows, I use a utility called HostMan, but you can simply edit your hosts file. Once done, all traffic to that domain will hit that IP.
> * Then you can go into your Caprover interface and change the domain to a new one.

Thank you for your reply and time. Can you elaborate it in detail with an example, if possible?
